src/openforms/js/components/formio_builder/translation.js

```diff
@@ -25,7 +25,9 @@ const LANGUAGES = getSupportedLanguages().map(([langCode]) => langCode);
  * the exact component type.
  * @type {string[]}
  */
-const ALWAYS_TRANSLATABLE_PROPERTIES = ['label', 'description', 'placeholder', 'tooltip'];
+const ALWAYS_TRANSLATABLE_PROPERTIES = ['label', 'description', 'placeholder', 'tooltip'].flatMap(
+  prop => [prop, 'basic.' + prop]
+);
```

Comment:
FormIO prepends the property names with 'basic.', because...

Reply:
Wait... This looks like something we've added... Which might be breaking things.